cfme/containers/container.py

```diff
@@ -58,3 +58,13 @@ def get_detail(self, *ident):
         """
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
+
+    @staticmethod
+    def get_container_names():
+        sel.force_navigate('containers_containers')
+        return map(lambda r: r.name.text, list_tbl.rows())
```

Comment:
This will not paginate so if there is more the 20 or so containers, it will give you incorrect values and there will be no errors, it will just straight up lie to you.